yat-lib/src/main/java/yat/android/sdk/apis/LootBoxTypeApi.kt

```diff
@@ -35,23 +34,22 @@ class LootBoxTypeApi : ApiClient() {
     }
 
     /**
-    * Create a new lootbox category
-    * NOTE: user must have scope &#x60;AdminLootBoxWrite&#x60;
-    * @param body  
-    * @return PublicLootBoxType
-    * @throws UnsupportedOperationException If the API returns an informational or redirection response
-    * @throws ClientException If the API returns a client error response
-    * @throws ServerException If the API returns a server error response
-    */
-    @Suppress("UNCHECKED_CAST")
+     * Create a new lootbox category
+     * NOTE: user must have scope &#x60;AdminLootBoxWrite&#x60;
+     * @param body
+     * @return PublicLootBoxType
+     * @throws UnsupportedOperationException If the API returns an informational or redirection response
+     * @throws ClientException If the API returns a client error response
+     * @throws ServerException If the API returns a server error response
+     */
     @Throws(UnsupportedOperationException::class, ClientException::class, ServerException::class)
-    fun createType(body: AdminNewLootBoxType) : PublicLootBoxType {
-        val localVariableBody: Any? = body
+    suspend fun createType(body: AdminNewLootBoxType): PublicLootBoxType {
```

Comment:
Have you generate all those methods using OpenAPI tools?

Reply:
Hm. I don't know, I just added the suspend modifier to the existing code

Reply:
But it's true that this code looked auto-generated because it has unnecessary imports and sometimes weird formatting

Reply:
You can change that editing the template